service/collector_test.go

```diff
@@ -255,8 +254,11 @@ receivers:
   otlp:
     protocols:
       grpc:
+
 exporters:
-  logging:
+  otlp:
+    endpoint: "localhost:4317"
```

Comment:
Can you explain how changing this fixed the need to skip the test on darwin?

Reply:
Because I am not using the logging exporter anymore which was the problem.